adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java

```diff
@@ -62,15 +62,18 @@
             .registerTypeAdapter(TokenCacheItem.class, new TokenCacheItemSerializationAdapater())
             .create();
 
-    private int getVersion() {
-        return version;
+    /**
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     */
+    @SuppressWarnings("unused")
```

Comment:
@heidijinxujia Were the unit tests failing before his changes? This was a private constructor, what has changed around the tests and causing the failure?